ReportingServicesTools/Functions/CatalogItems/Update-RsSubscription.ps1

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.

```

Comment:
Please add tests for the new function

Reply:
Finished the pester test and will be issuing a new PR this week